src/components/custom/_simple-steps.tsx

```diff
@@ -97,50 +93,33 @@ const ClientCard = styled.article<ClientCardProps>`
         }
     }
 `
-
 const LinkButtonWrapper = styled(Flex)`
     margin-top: 32px;
     text-align: center;
     justify-content: center;
 `
 
-const StyledLinkButton = styled(Button)`
-    height: 40px;
-    width: auto;
-    border-radius: 4px;
-    position: relative;
-    white-space: nowrap;
-`
-
-const SimpleSteps = ({ header, content, sign_up }: SimpleStepsProps) => {
+const SimpleSteps = ({ header, content }: SimpleStepsProps) => {
     const handleSignup = useHandleSignup()
-    const [is_logged_in] = useAuthCheck()
-    const { is_deriv_go } = usePlatformQueryParam()
 
     return (
         <StyledSection>
             <Container direction="column">
                 <TitleHeader align="center" as="h2" type="section-title">
-                    {header}
+                    {header && <Localize translate_text={header} />}
                 </TitleHeader>
             </Container>
             <StyledFlex wrap="wrap">
-                {content.map((item, idx) => {
+                {content.map(({ header, icon, text }) => {
```

Comment:
since content is optional (content?  according to your ts declaration ) can we use  to avoid errors 
```suggestion
                {content?.map(({ header, icon, text }) => {
```